Workspace build issue

When no custom SSL certificate is added, workspace container fails to build.

The error shown in console is the following:

```
Step 38/46 : RUN chmod -R 644 /etc/ssl/certificates/*
 ---> Running in 7e415667b33d
chmod: cannot access '/etc/ssl/certificates/*': No such file or directory
ERROR: Service 'php' failed to build: The command '/bin/sh -c chmod -R 644 /etc/ssl/certificates/*' returned a non-zero code: 1
make: *** [build] Error 1
```

--- a/docker/workspace/Dockerfile
+++ b/docker/workspace/Dockerfile
@@ -191,7 +191,11 @@ RUN usermod -u 1000 www-data \
 COPY ssl-certificates /etc/ssl/certificates/
 COPY ssh-keys/ /root/.ssh/
 
-RUN chmod -R 644 /etc/ssl/certificates/*
+
+RUN if [ $(ls -1 /etc/ssl/certificates/*.crt 2>/dev/null | wc -l) != 0 ]; then \
+    chmod -R 644 /etc/ssl/certificates/* \
+;fi
+
 RUN if [ $(ls -1 /root/.ssh/id_rsa* 2>/dev/null | wc -l) != 0 ]; then \
     chmod 600 /root/.ssh/id_rsa \
     && chmod 644 /root/.ssh/id_rsa.pub \

